src/components/app/BottomBox.css

```diff
@@ -42,3 +42,47 @@
   background: url(../../../res/img/svg/close-dark.svg) no-repeat center / 16px
     16px;
 }
+
+.sourceStatusOverlay {
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  display: flex;
+  width: max-content;
```

Comment:
I'm not sure, I can test what happens if I remove `width: max-content`.
It interacts with `right/left:0` in such a way that the remaining space is contributed by `margin-left/right`. If one of the margins is fixed and the other is auto, then the auto margin takes up the full remaining space. But since both are auto, they share the space evenly, and the element gets centered.

Reply:
Removing `width: max-content` makes it so that that the `left/right:0` pull the element to be wider, until it runs into the `max-width`. So I'll keep the width declaration.